src/test/javascript/integration/webhook_lifecycle_previous_spec.js

```diff
@@ -39,7 +39,7 @@ const callbackItems = [];
  * 2 - create a webhook on that channel with startItem=previous
  * 3 - start a server at the endpoint
  * 4 - post item into the channel
- * 5 - verify that the item are returned within delta time, incuding the second item posted in 2.
+ * 5 - verify that the item are returned within delta time, including the second item posted in 2.
```

Comment:
I remember you mentioning that it would be easier to have all our tests in one language. Would this be part of refactor we do or do we create separate Jira ticket for this?

Reply:
Yes, generally I think when refactoring a test it should be refactored to java, these tests however have nothing to do with this PR and refactoring them is well out of it's scope. I accidentally included these typo fixes in a commit and will remove it for now